ci-operator/step-registry/ipi/conf/gcp/filestore-workload-identity-federation-cleanup/ipi-conf-gcp-filestore-workload-identity-federation-cleanup-commands.sh

```diff
@@ -18,7 +18,7 @@ logger() {
 
 function run_command() {
     local CMD="$1"
-    echo "Running Command: ${CMD}"
+    echo "Running Command: ${CMD}" || true
```

Comment:
Suggest to follow similar as https://github.com/openshift/release/blob/master/ci-operator/step-registry/ipi/conf/gcp/filestore-workload-identity-federation/ipi-conf-gcp-filestore-workload-identity-federation-commands.sh#L23